packages/rum-core/src/domain/limitModification.ts

```diff
@@ -53,6 +53,10 @@ function set(object: unknown, path: string, value: unknown) {
   }
 }
 
+function isValidObject(object: unknown): object is { [key: string]: unknown } {
+  return typeof object === 'object' && object !== null
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this could be replaced with `getType(object) === 'object'`